.github/workflows/unitTests.yml

```diff
@@ -23,3 +23,4 @@ jobs:
         with:
           php_version: ${{ matrix.php-version }}
           configuration: tests/phpunit.xml
+          version: 9.5.28
```

Comment:
@narayana-plivo  According to the error messages on the Unit Tests workflow, the newer PHPUnit Versions (> 9.5) need to have PHP version >= 8.1. So should we fix PHPUnit version here to 9.5 or update the PHP versions used for unit tests?

Reply:
Its good.Few customers are using php 7